.github/workflows/deploy-eks.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
       
       - name: Test bundle deployment
         run: |
-          tox -vve test_bundle_deployment -- --model kubeflow --keep-models -vv -s
+          tox -vve test_bundle_deployment-1.7 -- --model kubeflow --keep-models -vv -s
```

Comment:
Is there a reason to keep 1.7 only?
This is going to be interesting. The `deploy-eks.yaml` file has a lot of pins to versions that our 1.7 release would use (e.g. microk8s and juju). I think that we should run either both of our supported versions or our latest/edge at least to keep this workflow relevant. This of course requires us to make this file more agnostic to the versions of different components, ensuring in the future we can just easily change the supported versions.

Reply:
No there isn't and there is actually a separate task about doing exactly what you said #801. I agree that we should run both of the supported versions. I hadn't thought of `latest/edge` TBH. That being said, I have already implemented this for AKS (see [changes in PR](https://github.com/canonical/bundle-kubeflow/pull/804/files#diff-d31ac3f5da3116ac2c1ebd29e6384c49fb4360e34ba6155825fcaae903b4b8d4)). This is under review and blocked by the fact that we do nohave access to AKS credentials, but the implementation is there.

Reply:
Perfect, let's track it in #801!